dmutils/errors.py

```diff
@@ -5,16 +5,17 @@
 
 def csrf_handler(e):
     """
-    Workaround for a bug in Flask 0.10.1.
-    CSRFErrors are caught under 400 BadRequest exceptions, so this heavy-handed solution
-     catches all 400s, and immediately discards non-CSRFError instances.
-
     :param e: CSRF exception instance
     :param session: Flask session instance
     :param request: Flask request instance
     :param logger: app logger instance
     :return: redirect to login with flashed error message
     """
+
+    # This is the remnant of a workaround for a bug in Flask 0.10.1. In that case, CSRFErrors
+    # are caught under 400 BadRequest exceptions, so this handler can be registered to catch
+    # all 400s, but will immediately discards non-CSRFError instances. Can be removed once
+    # we're absolutely certain nothing is still using Flask 0.10.1.
```

Comment:
I think we can remove it now. If the app has pulled in this version of utils it will have Flask 0.12, and `flask_init` will only supply this function with recognisable `CSRFError` instances (as opposed to, say, `BadRequest`).

Reply:
For clarity we could rename the `e` arg to `csrf_error`?

Reply:
:+1: